### PR TITLE
Studentnav

### PR DESCRIPTION
**Pull request by prassuresh108**
Added a student search screen and some styling to the faculty's opportunity creation screen.

### CODE REVIEW

**Review by nachobrothernic — APPROVED** [review]
LGTM

File: client/src/app/components/post_creation/posts.component.html [hunk 1]
```diff
@@ -1,73 +1,92 @@
...
-    <br>
-    <label> Deadline: <mat-form-field color="accent" appearance="fill">
+    <div id="websiteReqs" *ngIf="requirementsType == 0">
+
+      <label> 
+        Description: <br> 
+        <input class = "text-field" type="text" [(ngModel)]="description" name="description" required>
+        <br>
+      </label>
+      <br />
```
Comment (lukebushur) [inline client/src/app/components/post_creation/posts.component.html:32]: Try to be consistent with the `<br>` tags - use only `<br>`, as that is what's used in other components.

**Review by lukebushur — APPROVED** [review]
Other than the `<br>` tags, looks good to me!

File: client/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -29,6 +30,7 @@ const routes: Routes = [
   { path: "industry-dashboard", component: IndustryDashboardComponent, canActivate: [AuthGuard] },
   //This route has two URL parameters, one for projectID, and one for applicationID. It is used to access a specific applicant's data by the faculty
   { path: "application/:projectID/:applicationID", component: ViewApplicationComponent },
+  { path: "student-search-opps", component: StudentSearchOppsComponent, canActivate: [AuthGuard] },
```
Comment (stierney0505) [inline client/src/app/app-routing.module.ts:33]: Can you resolve the merge conflicts without removing any other routes on the front end?

**Review by stierney0505 — APPROVED** [review]

**Pull request merged**